### DISCUSSION TITLE
customizing Dashboard in Elsa 1

### DISCUSSION THREAD

**Discussion by akghasemi** [elsa-workflows/elsa-core#1448] [Q&A]
Hi,
Is there anyway that I can customize dashboard in Elsa 1. it seems everything is inside the library and I can't override the dashboard page or change side bar menu and it seems the paging doesn't work either

**Comment by sfmskywalker** [comment]
The Elsa 1 dashboard is an early prototype that doesn't support much customization and development on it has been discontinued in favor of Elsa 2. 

If you want to continue using Elsa 1 then you might want to just copy the Dashboard MVC area's source code into your own application and make changes as necessary.
